src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService user = new UserService();
```

Comment:
it should be class-level variable (not method)
in this case - you create `userService` object each time you call `login` method
Wrong name of variable
```suggestion
        UserService userService = new UserService();
```